source/scpd/types/Stellar_types.d

```diff
@@ -44,6 +44,6 @@ unittest
     static import agora.crypto.Schnorr;
 
     static assert(agora.crypto.Hash.Hash.sizeof == Hash.sizeof);
-    static assert(agora.crypto.Hash.Hash.sizeof == NodeID.sizeof);
+    static assert(ulong.sizeof == NodeID.sizeof);
```

Comment:
You can remove the line, the other ones are to ensure binary compatibility, but since we use `ulong` directly this is not necessary.

Reply:
Removed